.github/workflows/auto-test.yml

```
@@ -78,7 +78,10 @@ jobs:
 
   e2e-test:
     needs: [ ]
-    runs-on: ARM64
+    strategy:
+      matrix:
+        os: [ ARM64, ubuntu-24.04-arm ]
+    runs-on: ${{ matrix.os }}
```

https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners

Above link does not list `ARM64` anymore.
I think we can savely switch this, or am I missing something?

```suggestion
    runs-on: ubuntu-24.04-arm
```